catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/DeleteLookDialogTest.java

```diff
@@ -115,11 +114,11 @@ public void deleteLookDialogTest() {
 	@Category({Cat.AppUi.class, Level.Smoke.class})
 	@Test
 	public void cancelDeleteLookDialogTest() {
-		openActionBarOverflowOrOptionsMenu(getInstrumentation().getTargetContext());
+		openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getInstrumentation().getTargetContext());
 		onView(withText(R.string.delete)).perform(click());
 
 		onRecyclerView().atPosition(1)
-				.performCheckItem();
+				.performCheckItemClick();
 
 		onView(withId(R.id.confirm)).perform(click());
 
```

Comment:
I think there is a test missing here to check the checkbox, deleteSoundDialogTest and deleteSpriteDialogTest do include the test, please add it here as well,

Reply:
Also please fix static analysis warnings